lib/route.method.js

```diff
@@ -139,6 +139,15 @@ RouteMethod.prototype.validate = function (validateFunc) {
     return this;
 };
 
+RouteMethod.prototype.notAllowed = function () {
+    this.options.notAllowed = true;
+    return this;
+};
+
+RouteMethod.prototype.isAllowed = function () {
```

Comment:
A bit confused by this - disable/Enable is pretty clear to me; if you'd called this DisallowHTTPRoute, then this would be clearer... Maybe we need to chat about this one in real-time.